game-core/src/test/java/games/strategy/triplea/attachments/TriggerAttachmentTest.java

```diff
@@ -33,23 +37,29 @@
 @ExtendWith(MockitoExtension.class)
 class TriggerAttachmentTest {
 
-  private final Set<TriggerAttachment> satisfiedTriggers = new HashSet<>();
   @Mock
   private IDelegateBridge bridge;
   @Mock
   private IDelegateHistoryWriter historyWriter;
 
   @BeforeEach
-  void setUp() {
+  void setUp() throws MutableProperty.InvalidValueException {
     final GameData gameData = new GameData();
 
+    when(bridge.getData()).thenReturn(gameData);
+    when(bridge.getHistoryWriter()).thenReturn(historyWriter);
+  }
+
+  @Test
+  void testTriggerProductionFrontierEditChange() {
+
```

Comment:
The newline after the declaration is odd. I wouldn't complain normally because it's probably just a style preference (we had some discussions about style guides in the past with a lot of mixed opinions, not going to start the debate again) but while you change the test file anyways you could easily just remove both lines in this file 👍